lib/src/platform/src/realtime/realtime.dart

```diff
@@ -60,41 +55,49 @@ class Realtime extends PlatformObject {
   // The _connection instance keeps a reference to this platform object.
   late final Connection _connection;
 
-  /// Provides access to the underlying Connection object
-  ///
-  /// https://docs.ably.com/client-lib-development-guide/features/#RTC2
+  /// A [Connection] object.
   Connection get connection => _connection;
 
-  /// a custom auth object to perform authentication related operations
-  /// based on the [options]
-  ///
-  /// https://docs.ably.com/client-lib-development-guide/features/#RSC5
+  /// An [Auth] object.
   // Auth? auth;
 
-  /// [ClientOptions] indicating authentication and other settings for this
-  /// instance to interact with ably service
+  /// @nodoc
+  /// A [ClientOptions] object used to configure the client connection to Ably.
   late ClientOptions options;
 
-  /// a push object interacting with Push API
-  /// viz., subscribing for push notifications, etc
+  /// A [Push] object.
   late Push push;
 
   late RealtimeChannels _channels;
 
-  /// Provides access to the underlying [RealtimeChannels]
-  ///
-  /// https://docs.ably.com/client-lib-development-guide/features/#RTC3
+  /// A [Channels] object.
   RealtimeChannels get channels => _channels;
 
-  /// closes the [connection]
+  /// Calls [Connection.close] and causes the connection to close, entering the
+  /// closing state. Once closed, the library will not attempt to re-establish
+  /// the connection without an explicit call to [Connection.connect].
   Future<void> close() async => invoke(PlatformMethod.closeRealtime);
 
-  /// connects to [connection]
+  /// Calls [Connection.connect] and causes the connection to open, entering the
+  /// connecting state. Explicitly calling [Connection.connect] is unnecessary
+  /// unless the [ClientOptions.autoConnect] property is disabled.
   Future<void> connect() async => invoke<void>(PlatformMethod.connectRealtime);
 
-  /// creates and sends a raw request to ably service
+  /// Makes a REST request to a provided [path] using a [method], such as `GET`,
```

Comment:
I think we could add an empty line after the first sentence to match https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary

Reply:
Done in 6c4b6b04df275b6136936cfec3d2b520b0477a55